src/components/rocketData.js

```diff
@@ -0,0 +1,463 @@
+/* eslint-disable */
```

Comment:
- [ ] Optional 

Please kindly fix the linter error. 
Disabling eslint rules for an entire file is generally not recommended, as it can lead to potential issues and decrease code quality. It's better to address and fix any issues reported by eslint to ensure clean and maintainable code.